src/xenia/kernel/xam/xam_nui.cc

```diff
@@ -35,23 +57,298 @@ struct X_NUI_DEVICE_STATUS {
 };
 static_assert(sizeof(X_NUI_DEVICE_STATUS) == 24, "Size matters");
 
-void XamNuiGetDeviceStatus_entry(pointer_t<X_NUI_DEVICE_STATUS> status_ptr) {
+// Get
+dword_result_t XamNuiGetDeviceStatus_entry(
+    pointer_t<X_NUI_DEVICE_STATUS> status_ptr) {
+  /* Notes:
+     - it does return a value that is not always used
+     - returns values are X_ERROR_SUCCESS, 0xC0050006, and others
+     - 1) On func start *status_ptr = 0, status_ptr->unk1 = 0, status_ptr->unk2
+     = 0, and status_ptr->status = 0
+     - 2) calls XamXStudioRequest(6,&var <- = 0);
+     - if return is greater than -1 && var & 0x80000000 != 0 then set
+     status_ptr->unk1 = 0xC000009D, status_ptr->unk2 = 0xC000009D, and
+     status_ptr->status = status_ptr[3] = 0x20
+     - lots of branching functions after
+  */
+
   status_ptr.Zero();
-  status_ptr->status = 0;  // Not connected.
+  status_ptr->status = cvars::Allow_nui_initialization ? true : false;
```

Comment:
Won't ``status_ptr->status = cvars::Allow_nui_initialization`` suffice? If not then it can stay as it is